app/src/main/java/com/keylesspalace/tusky/util/Theme.kt

```diff
@@ -0,0 +1,13 @@
+package com.keylesspalace.tusky.util
+
+import android.content.Context
+import android.util.TypedValue
+import androidx.annotation.AttrRes
+import androidx.annotation.ColorInt
+
+@ColorInt
+fun Context.getColorByAttribute(@AttrRes attr: Int): Int {
+    val typedValue = TypedValue()
+    theme.resolveAttribute(attr, typedValue, true)
+    return typedValue.type
+}
```

Comment:
Maybe refactor ThemeUtils to Kotlin and put this there as well so all the helper functions are in one place?

Reply:
can do!